publisher/product/ontology/build-about.sh

```diff
@@ -48,11 +40,11 @@ __HERE__
     "${JENA_ARQ}" \
       --data="${tmpAboutFileProd}" \
       --query="${SCRIPT_DIR}/lib/echo.sparql" \
-      --results=RDF > "${tag_root}/LoadFIBOProd.rdf"
+      --results=RDF > "${tag_root}/Load${ONTPUB_FAMILY}Prod.rdf"
```

Comment:
Should be uppercase:
`--results=RDF > "${tag_root}/Load${ONTPUB_FAMILY^^}Prod.rdf"`